src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,145 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int INCREMENT = 1;
+    private static int arraySize = 10;
+    private Object[] arrayList = new Object[arraySize];
```

Comment:
use constructor to init fields
```suggestion
    private T[] elements;
```